File: payload.c
#include <stdbool.h>
#include <stdint.h>
#include <stdio.h>
#include <wchar.h>
#include <windows.h>
#include <winternl.h>

int main(void) {
  printf("Test %d %s\n", 4, "str");
  WinExec("calc.exe", SW_SHOWNORMAL);
  return 0;
}

File: payload.rs
#![no_main]
use windows_sys::{
    core::*, Win32::System::Threading::*, Win32::UI::WindowsAndMessaging::*,
};
use libc::*;

#[no_mangle]
fn main() -> () {
    unsafe {
        WinExec(s!("calc.exe"), SW_SHOWNORMAL.try_into().unwrap());
        printf(s!("Test\n") as *const i8);
    }
}
